build.sbt

```diff
@@ -217,7 +217,7 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
       "org.typelevel" %%% "munit-cats-effect" % "2.0.0-M3" % Test,
       "org.typelevel" %%% "discipline-munit" % "2.0.0-M3" % Test
     ),
-    tlJdkRelease := Some(8),
+    tlJdkRelease := Some(9),
```

Comment:
Is this the right way to ensure the class compiles?

Reply:
Actually this will generate bytecode that is not compatible with Java 8 ... actually we should set it to `None` 🙃